docs/en/introduction.md

```diff
@@ -0,0 +1,99 @@
+# Introduction
+
+> In this chapter, we will use the Ribir syntax to write some simple examples. You only need to understand the general idea, no need to go into details. We will introduce it in detail in the following chapters.
```

Comment:
You only need to understand the general idea; there is no need to go into details.

Reply:
You just need to understand the general idea, you don't need to go deeper.